### PR TITLE
adding QIE10 ZDC digis to 2018 HI run

### PR DESCRIPTION
**Pull request by BetterWang**
adding QIE10 ZDC digis to 2018 HI run, pp_on_AA_2018.
This will add 50 ZDC/RPD channels with 8TS each.

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by BetterWang** [comment]
@mandrenguyen HI software contact person.

File: RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py [hunk 1]
```diff
@@ -79,7 +79,8 @@ def _updateOutput( era, outputPSets, commands):
 phase2_hgcal.toModify( RecoLocalCaloAOD, outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_HGCalRecHit_*_*','keep *_hgcalLayerClusters_*_*'] )
 
 from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
-pA_2016.toModify( RecoLocalCaloAOD.outputCommands,
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+(pA_2016|pp_on_AA_2018).toModify( RecoLocalCaloAOD.outputCommands,
```
Comment (perrotta) [inline RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py]: All other modifications of the event content for rhe pp_on_AA_2018 era modify at the same time the AOD, the RECO and the FEVT event content, see e.g. 
https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_10_3_X_2018-08-31-2300&_filestring=EventContent&_string=pp_on_AA_2018

Here you only add ZDC digis to the AOD event content: is this really what you plan for that era?

Comment (BetterWang) [reply RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py]: You are right. I plan to add ZDC digis to AOD, RECO and FEVT.
Just pushed another commit.

File: RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py [hunk 2]
```diff
@@ -86,3 +86,13 @@ def _updateOutput( era, outputPSets, commands):
...
                                                                      ])
                   )
+
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+for ec in [RecoLocalCaloAOD.outputCommands, RecoLocalCaloRECO.outputCommands, RecoLocalCaloFEVT.outputCommands]:
+	pp_on_AA_2018.toModify( ec,
+                  func=lambda outputCommands: outputCommands.extend(['keep *_zdcreco_*_*',
+                                                                     'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                                                     'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                                                     'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*'
```
Comment (slava77) [inline RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py]: it seems to me that these are already in the standard RECO and FEVT outputs.

Comment (BetterWang) [reply RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py]: You are right. They are defined in the same file.
FEVT: https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py#L20
RECO: 
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py#L39

Should I revert to the previous commit?

Comment (perrotta) [reply RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py]: Indeed... I did not notice it, sorry.
Yes, either: revert the last commit; or remove from standard RECO and FEVT; or add also to the standard AOD (and remove the customization at all).
I think that reverting the last commit is the simplest (and most obviously back compatible) way of dealing with it.

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
Just for the record, this is the size added to the AOD output:

```
zdcreco: (ZDCRecHitsSorted) 58.0286 38.4857
castorDigis: (ZDCDataFramesSorted) 59.7714 40.2286
hcalDigis: (ZDCDataFramesSorted) 59.3714 39.8286
hcalDigis:ZDC (QIE10DataFrameHcalDataFrameContainer) 87.3143 42.2571
```

**Comment by perrotta** [comment]
+1
- QIE10 ZDC digis added to the AOD event content for the 2018 HI campaigns (they were already included in the RECO and FEVT event contents)
- Jenkins tests pass

**Comment by fabiocos** [comment]
+1

**Pull request merged**